zerver/webhooks/dialogflow/doc.md

```diff
@@ -0,0 +1,23 @@
+Know the results of your queries on your dialogflow app by PM in Zulip !
+
+{!create-bot-construct-url.md!}
+
+url structure should be like this:
+
+`{{api_url}}?api_key=BOT'S_API_KEY&email=foo@example.com`
+
```

Comment:
Okay, so the same comments I posted on your Beeminder webhook PR apply here :) Also, don't use `\` for breaking lines please